common/app/views/fragments/email/support.scala.html

```diff
@@ -1,5 +1,7 @@
 @(page: model.Page)(implicit request: RequestHeader)
 
+@componentIdPart = @{ if (page.metadata.isFront) request.path.toLowerCase.replace("/", "_").replaceFirst(".email(json|txt)", "").filter(_.toString.matches("[a-z_]")) else "_email_article" }
```

Comment:
Can this be pulled out into a helper function? I'm not sure what `componentIdPart` means.